### ISSUE TITLE
Signalling for rate-limits for a client?

### ISSUE THREAD

**Issue by gloinul** [ietf-wg-masque/draft-ietf-masque-quic-proxy#39]
The draft do discuss the aspect that the proxy may rate limit a client. Should there actually be explicit signalling of these rates to the client. 

I also assume these parameters are on client level and thus apply across the different UDP flows.

**Comment by tfpauly** [comment]
The rate-limiting here is referring to the number of CONNECT or CONNECT-UDP requests the proxy allows for a given client. This would translate into delayed or rejected requests, as a normal HTTP proxy behavior.

**Comment by LPardue** [comment]
Yeah. I think there is nothing special to do here, implementations or deployments can leverage HTTP layer mechanisms if they care.

**Comment by gloinul** [comment]
No, I meant any limits in bits or number of packets.

**Comment by LPardue** [comment]
Unless something like this already exists for CONNECT in HTTP/1.1 or HTTP/2 I don't know that we need to do anything.

**Comment by tfpauly** [comment]
@gloinul I think having strict limits on the number of packets or bytes that go through a proxy connection could be enforced by a proxy closing a given CONNECT request stream if it violates a policy. If you want explicit signaling about expected rates or datagram flow control, that'd need to be another separate extension.

**Comment by tfpauly** [comment]
I believe that there isn't anything to do here — I agree with Lucas's points. Please re-open if you disagree!
